tests/integration/test_es.py

```diff
@@ -102,29 +102,21 @@ def test_get_compatible_version_for_domain(self, es_client, opensearch_domain):
         # The default version is the latest version, which is not compatible with any previous versions
         assert len(versions) == 0
 
-    def test_create_domain(self, es_client):
-        domain_name = f"es-domain-{short_uid()}"
-        try:
-            es_client.create_elasticsearch_domain(DomainName=domain_name)
-
-            response = es_client.list_domain_names(EngineType="Elasticsearch")
-            domain_names = [domain["DomainName"] for domain in response["DomainNames"]]
+    def test_create_domain(self, es_client, opensearch_create_domain):
+        es_domain = opensearch_create_domain(EngineVersion="Elasticsearch_7.10")
```

Comment:
there's the `ELASTICSEARCH_DEFAULT_VERSION` which i think we should use here (and line 112)